index.bs

```diff
@@ -9836,10 +9836,17 @@ Attributes</h5>
 		When an exception is thrown from the processor's
 		<code>constructor</code>, <code>process</code> method,
 		or any user-defined class method, the processor will
-		<a>queue a task</a> on the control thread to fire
-		{{onprocessorerror}} event to the node. Note that
-		once an exception is thrown, the processor will output
-		silence throughout its lifetime.
+		<a>queue a task</a> to fire an
```

Comment:
What if the exception is handled? Is there a specific formulation to refer to this case?

Reply:
A good point. I think we should only deliver the exception to the node when it is "unhandled". We could throw two exceptions on both sides, but it seems redundant and not particularly useful.